### PR TITLE
You can use this email address and commits will be linked correctly

### PR DESCRIPTION
**Pull request by julien-c**
cc @pierrci

### CODE REVIEW

File: HuggingPics.ipynb [hunk 1]
```diff
@@ -483,7 +482,7 @@
         "token = HfApi().login(username=hf_username, password=hf_password)\n",
         "del hf_password\n",
         "model_url = HfApi().create_repo(token=token, name=model_id, exist_ok=True)\n",
-        "model_repo = Repository(\"./model\", clone_from=model_url, use_auth_token=token, git_email=git_email, git_user=hf_username)\n",
+        "model_repo = Repository(\"./model\", clone_from=model_url, use_auth_token=token, git_email=f"{hf_username}@users.noreply.huggingface.co", git_user=hf_username)\n",
```
Comment (nateraw) [inline HuggingPics.ipynb:485]: 🔥 nice

**Review by nateraw — APPROVED** [review]
Nice! The email there bothered me.

**Comment by nateraw** [comment]
This broke the notebook, so I reverted it after merge. Going to just commit directly to main w/ Colab.

**Comment by julien-c** [comment]
Oops yeah might not have been the best idea to update an .ipynb manually

**Pull request merged**